src/main/java/org/candlepin/sync/SyncUtils.java

```diff
@@ -63,34 +55,9 @@ File makeTempDir(String baseName) throws IOException {
     }
 
     @Inject
-    public SyncUtils(Configuration config) {
+    public SyncUtils(Configuration config,
+        @Named("SyncUtilsObjectMapper") ObjectMapper objectMapper) {
         this.config = config;
-
-        this.mapper = new ObjectMapper();
-        AnnotationIntrospector primary = new JacksonAnnotationIntrospector();
-        AnnotationIntrospector secondary = new JaxbAnnotationIntrospector(this.mapper.getTypeFactory());
-        AnnotationIntrospector pair = new AnnotationIntrospectorPair(primary, secondary);
-
-        this.mapper.setAnnotationIntrospector(pair);
-        this.mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
-
-        // Add support for new JDK8 features
-        this.mapper.registerModule(new Jdk8Module());
-        mapper.registerModule(new JavaTimeModule());
-
-        // Filter specific things we do not want exported:
-        SimpleFilterProvider filterProvider = new SimpleFilterProvider();
-        filterProvider.setFailOnUnknownId(false);
-
-        this.mapper.setFilterProvider(filterProvider);
-
-        if (config != null) {
-            this.mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES,
-                config.getBoolean(ConfigProperties.FAIL_ON_UNKNOWN_IMPORT_PROPERTIES));
-        }
-    }
-
-    public ObjectMapper getObjectMapper() {
-        return this.mapper;
+        this.mapper = objectMapper;
```

Comment:
The removal of the `getObjectMapper` method invalidates the need for the mapper at all in this class.